extension-manifest-v3/src/store/options.js

```diff
@@ -83,6 +83,7 @@ const Options = {
     ...(__PLATFORM__ === 'opera' ? { serpShownAt: 0, serpShown: 0 } : {}),
   },
   installDate: '',
+  installRandomNumber: 0,
```

Comment:
Please use `installRandomId` or `installRandomID` - it's obvious that it's a number... ;) We don't use `installDateString` ;)